### PR TITLE
[foreman] Use psql-msgpack-decode wrapper for dynflow >= 1.6

### PR DESCRIPTION
**Pull request by pmoravec**
In dynflow >=1.6.3, dynflow* tables in postgres are encoded by
msgpack which makes plain CSV dumps unreadable. In such a case,
psql-msgpack-decode wrapper tool from dynflow-utils (of any
version) must be used instead of the plain psql command.

Resolves: #2830

Signed-off-by: Pavel Moravec <pmoravec@redhat.com>

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [X] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

File: sos/report/plugins/foreman.py [hunk 1]
```diff
@@ -244,8 +244,17 @@ def setup(self):
             self.add_cmd_output(_cmd, suggest_filename=table, timeout=600,
                                 sizelimit=100, env=self.env)
 
+        # dynflow* tables on dynflow >=1.6.3 are encoded and hence in that
+        # case, psql-msgpack-decode wrapper tool from dynflow-utils (any
+        # version) must be used instead of plain psql command
+        dynutils = 'dynflow-utils' in \
+            self.policy.package_manager.get_pkg_list().keys()
```
Comment (TurboTurtle) [inline sos/report/plugins/foreman.py:251]: Is `self.is_installed('dynflow-utils')` not sufficient here?

Comment (pmoravec) [reply sos/report/plugins/foreman.py:251]: Ah right. It was a relic of previous testing "dynutils of version .." which is not required.

**Review by TurboTurtle — APPROVED** [review]

**Comment by adamruzicka** [comment]
dynflow-utils is already available in nightly foreman repos http://yum.theforeman.org/nightly/el7/x86_64/dynflow-utils-1.6.3-1.el7.x86_64.rpm . For testing it can be installed anywhere, it has no dependencies and contains only a static binary and a shell script

**Comment by pmoravec** [comment]
> dynflow-utils is already available in nightly foreman repos http://yum.theforeman.org/nightly/el7/x86_64/dynflow-utils-1.6.3-1.el7.x86_64.rpm . For testing it can be installed anywhere, it has no dependencies and contains only a static binary and a shell script

Thanks. The package and binary works great on my reproducer of this PR.

**Pull request merged**